app/templates/auth/login.html

```diff
@@ -12,8 +12,10 @@
     {% endif %}
         <p class="banner-message">
             {% if message == 'no_account' %}
-            Sorry, we couldn't find a supplier account with that username
-            and password.
+            Sorry, we couldn't log you in with that username and password.
```

Comment:
Is it worth breaking up this text a bit do you think?  Maybe a line break or two after the first sentence?

![screen shot 2015-07-06 at 09 34 28](https://cloud.githubusercontent.com/assets/6525554/8518484/5058b908-23c2-11e5-8242-cdb619988157.png)